projects/jansson/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "https://github.com/akheron/jansson"
+primary_contact: "git@cfware.com"
+auto_ccs:
+  - "cmeister2@gmail.com"
+sanitizers:
```

Comment:
Any reason why `memory` isn't here?

Reply:
No immediate reason; from https://github.com/google/oss-fuzz/blob/master/docs/new_project_guide.md#sanitizers-optional it seems to indicate that enabling `memory` can cause false positives. The flag it's hidden behind is setting `experimental: True`, so I thought it was still experimental.